.claude/skills/improve-card-generator/examples/debug-parser.ts

```diff
@@ -0,0 +1,16 @@
+import { parseAbilityText } from "../../src/parser";
+
+// Replace this with the text you want to debug
+const text =
```

Comment:
Let's receive this as an argument, so it's easier to call it. It can receive an array or a single text